village/src/components/Smurfs.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
-
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
+  
```

Comment:
All of the properties being passed to the `Smurf` component are basically the entire `smurf` element returning from map. Instead of passing each prop individually, you can (1) `<Smurf {...smurf} key={smurf.id} />` or (2) `<Smurf smurf={smurf} key={smurf.id}/>`. Then in the `Smurf` component you would call (1) props._property-name_ or (2) props.smurf._property-name_.